sdk/openai/azure-ai-openai/README.md

```diff
@@ -286,6 +290,44 @@ for (ImageLocation imageLocation : images.getData()) {
 
 For a complete sample example, see sample [Image Generation][sample_image_generation].
 
+### Audio Transcription
+The OpenAI service starts supporting `audio transcription` since model `Whisper`. 
```

Comment:
I would word this a little bit differently. How about "with the introduction of `Whisper` models" ?